### ISSUE TITLE
The `.err` style in `autumn.css` fails Chrome's contrast audit

### ISSUE THREAD

**Issue by ghost** [mpchadwick/pygments-high-contrast-stylesheets#2]
The error style in `autumn.css`
```
.highlight .err { color: #ec0505; background-color: #FFAAAA } /* Error */
```
has contrast ratio only 2.52:1, so fails AA. (See https://webaim.org/resources/contrastchecker/?fcolor=EC0505&bcolor=FFAAAA.)

**Comment by mpchadwick** [comment]
Thanks @barnes1463. Took a look at this and found that `tools/make-stylesheet` is not accounting for selectors that override `background-color`. Will work on a fix for this where I can.

**Comment by mpchadwick** [comment]
@barnes1463 I [updated](https://github.com/mpchadwick/pygments-high-contrast-stylesheets/commit/9afc786a90cdab5c726280c01b77510c9b32d55a#diff-8abdab0f6533094ca12dfdc59303090a) `tools/make-stylesheet` to check for selector level background colors and regenerated all the stylesheets. Should be good now:

https://github.com/mpchadwick/pygments-high-contrast-stylesheets/blob/5c637dd0c26aabbcd748701f96f69bbcbf5b0744/autumn.css#L4

https://webaim.org/resources/contrastchecker/?fcolor=A00000&bcolor=FFAAAA
